src/util.ts

```diff
@@ -598,7 +598,7 @@ export class Util {
       const callback =
         typeof optionsOrCallback === 'function' ? optionsOrCallback : undefined;
 
-      const onAuthenticated = (
+      const onAuthenticated = async (
```

Comment:
It's a little suspicious that this was switched to an async method, but I don't see an `await` anywhere getting added.  Where is this function called from?

Reply:
This was just to enable me to use an await inside to get the project ID, without using the whole promise/callback thing.

Reply:
Sure, but now when this method is called, it won't execute synchronously.  Where we invoke `onAuthenticated`, I would expect to now require an `await`?

Reply:
Sorry, that was definitely a mistake! I've made an update to consolidate some code and embrace the promise chain :)